extraction/named_entity/ner.py

```diff
@@ -1,134 +0,0 @@
-import abc
```

Comment:
Please don't delete this file.

Reply:
I see the problem. I will create another pull request.